videointelligence/tests/system.py

```diff
@@ -73,61 +84,72 @@ def test_annotate_video(self):
 
 
 @unittest.skipUnless(
-    OUTSIDE_PROJECT_API_KEY,
-    "GOOGLE_CLOUD_TESTS_VPCSC_OUTSIDE_PERIMETER_PROJECT_API_KEY not set in environment.",
+    CREDENTIALS_FILE,
+    "GOOGLE_APPLICATION_CREDENTIALS not set in environment.",
 )
 class TestVideoIntelligenceClientVpcSc(VideoIntelligenceSystemTestBase):
```

Comment:
If the tests do not use the client, maybe a different location would be more appropriate?

Reply:
Need them here for VPC SC team, they use to run there kokoro flow.

Reply:
Thank you for the clarification over chat earlier. I read the requirements list from the VPCSC. I *think* they want you to write normal system tests that exercise the client. The monitoring tests do this - checking that a call fails/succeeds in the expected way in the different environments. https://github.com/googleapis/google-cloud-python/blob/7572488283cb3c54374101de251571e86fa51a5d/monitoring/tests/system/test_vpcsc.py#L56-L62

@bmccutchon, would you be able to clarify the requirements?

Reply:
I agree that, for the most part, we want tests that exercise the client (because we want to be sure that this works the way our users will be using it). Moreover, in this case, headers like `X-User-IP` are ignored as untrusted, so there's probably no reason not to use the client.

Reply:
The client does not allow use of the operations API which is needed in this test.